### PR TITLE
Check apache status

### PR DESCRIPTION
**Pull request by hanynowsky**
Added check-apache-status.rb plugin

### CODE REVIEW

File: bin/check-apache-status.rb [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+require 'sensu-plugin/check/cli'
+require 'open3'
+require 'net/http'
+require 'net/https'
+
+#
+# Check  Apache Status
+#
+class CheckApacheStatus < Sensu::Plugin::Check::CLI
+  option :busy, long: '--busy BUSYWORKERS', short: '-b BusyWorkers', description: 'Busy workers threshold', proc: proc(&:to_i), default: 150, required: false
```
Comment (majormoses) [inline bin/check-apache-status.rb:35]: could you split these options into multiple lines? This is a bit hard to read, something like:
```ruby
  option :busy,
         long: '--busy BUSYWORKERS',
         short: '-b BusyWorkers',
         description: 'Busy workers threshold',
         proc: proc(&:to_i),
         default: 150,
         required: false
```

File: bin/check-apache-status.rb [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+  option :user, short: '-user USER', long: '--user USER', description: 'User if HTTP Basic is used'
+  option :password, short: '-password USER', long: '--password USER', description: 'Password if HTTP Basic is used'
+  option :secure, short: '-s', long: '--secure', description: 'Use SSL'
+
+  # Acquire Apache mod_status
+  def acquire_mod_status
+    http = Net::HTTP.new(config[:host], config[:port])
+    if config[:secure]
+      http = Net::HTTP.new(config[:host], config[:secureport])
+      http.verify_mode = OpenSSL::SSL::VERIFY_NONE
```
Comment (majormoses) [inline bin/check-apache-status.rb:52]: can we make this an option? I would hate to blindly turn off verification for everyone and would default to not turning it off.

File: bin/check-apache-status.rb [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+    end
+    req = Net::HTTP::Get.new(config[:path])
+    if !config[:user].nil? && !config[:password].nil?
+      req.basic_auth config[:user], config[:password]
+    end
+    res = http.request(req)
+    case res.code
+    when '200'
+      res.body
+    else
```
Comment (majormoses) [inline bin/check-apache-status.rb:63]: Think you need to add a case for following redirects?

**Review by majormoses — COMMENTED** [review]
Overall looks good could you make the changes proposed and if possible add a test?

File: CHANGELOG.md [hunk 4]
```diff
@@ -5,6 +5,10 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 
 ## [Unreleased][unreleased]
 
+## [0.0.6] - 2016-11-28
```
Comment (majormoses) [inline CHANGELOG.md:8]: please remove the version, we will bump when we release.

File: bin/check-apache-status.rb (hunk 2 above)
Comment (hanynowsky) [reply bin/check-apache-status.rb:52]: I see. Okay then.

File: bin/check-apache-status.rb (hunk 1 above)
Comment (hanynowsky) [reply bin/check-apache-status.rb:35]: OK.

File: CHANGELOG.md (hunk 4 above)
Comment (hanynowsky) [reply CHANGELOG.md:8]: ok

File: bin/check-apache-status.rb (hunk 3 above)
Comment (hanynowsky) [reply bin/check-apache-status.rb:63]: Same as the check-http? Have a suggestion?

Comment (majormoses) [reply bin/check-apache-status.rb:63]: something like setting a max number of redirects and a case statement something like this:
```
when Net::HTTPRedirection then
  location = response['Location']
  # rerun the request here with the location
end
```

Comment (hanynowsky) [reply bin/check-apache-status.rb:63]: Sounds obvious. Thanks.

**Comment by majormoses** [comment]
@hanynowsky are you planning on seeing this through to completion?

**Comment by majormoses** [comment]
closing due to inactivity, please reply back if you would like to see this through so I can re-open.

**Pull request closed without merging**